lib/web_ui/test/golden_tests/engine/canvas_draw_image_golden_test.dart

```diff
@@ -346,8 +346,176 @@ void main() async {
     await _checkScreenshot(rc, 'draw_text_composite_order_below',
         maxDiffRatePercent: 1.0);
   });
+
+  // Creates a picture
+  test('Paints nine slice image', () async {
+    Rect region = const Rect.fromLTWH(0, 0, 500, 500);
+    EnginePictureRecorder recorder = EnginePictureRecorder();
+    final Canvas canvas = Canvas(recorder, region);
+    Image testImage = createNineSliceImage();
+    canvas.clipRect(Rect.fromLTWH(0, 0,420, 200));
+    canvas.drawImageNine(testImage, Rect.fromLTWH(20, 20, 20, 20),
+        Rect.fromLTWH(20, 20, 400, 400), Paint());
+    Picture picture = recorder.endRecording();
+
+    final SurfaceSceneBuilder builder = SurfaceSceneBuilder();
+    builder.addPicture(Offset(0, 0), picture);
+
+    // Wrap in <flt-scene> so that our CSS selectors kick in.
+    final html.Element sceneElement = html.Element.tag('flt-scene');
+    try {
+      sceneElement.append(builder.build().webOnlyRootElement);
+      html.document.body.append(sceneElement);
+      await matchGoldenFile('draw_nine_slice.png',
+          region: region, maxDiffRatePercent: 0);
+    } finally {
```

Comment:
nit: "write: true", and also updating flutter/goldens

Reply:
See golden file PR this is waiting on.